.angular-cli.json

```diff
@@ -21,7 +21,8 @@
             "styles.scss",
             "theme.scss",
             "../node_modules/bootstrap/dist/css/bootstrap.min.css",
-            "../node_modules/dragula/dist/dragula.min.css"
+            "../node_modules/dragula/dist/dragula.min.css",
```

Comment:
check dragula library is using now or not. if not remove it. from all and packge.json too